### DISCUSSION TITLE
Use specific C++ compiler when using cppimport

### DISCUSSION THREAD

**Discussion by yentingl1** [tbenthompson/cppimport#92] [General]
Is it possible to ask cppimport to use specific c++ compiler during the compile/import process? cppimport works fine when I attempted to import a module which contains cpp files in Ubuntu, but it failed to import the exact same module in Mac. It looks like clang cannot compile the cpp files but the g++/c++ on Ubuntu had no issue. Thinking maybe forcing cppimport to use g++/c++ on Mac might work...

**Comment by tbenthompson** [comment]
Hi! cppimport is built on top of distutils so you should be able to set the compiler in the same way that you would with distutils: https://stackoverflow.com/questions/16737260/how-to-tell-distutils-to-use-gcc

**Reply by yentingl1** [reply]
Thanks a lot! It worked.
